Source/Core/Editor/Configuration/RuntimeConfiguratorEditor.cs

```diff
@@ -202,27 +204,47 @@ private static string GetProcessNameFromPath(string path)
 
         private void DrawProcessSelectionDropDown()
         {
-            int index = 0;
+            if (processDisplayNames[0] == dummyProcessName)
+            {
+                Debug.LogError("No process found. Did you delete or manually rename them?");
```

Comment:
Any reason it's "them" instead of "it"? It's true that many processes could have been deleted, but only one would have been selected.

```suggestion
                Debug.LogError("No process found. Did you delete or manually rename it?");
```

Reply:
For `if (processDisplayNames[0] == dummyProcessName)` it means that there is no process at all I thought it would be more likely that its more then one.

Reply:
@VaLiuM09 do you agree that this would be better?
```suggestion
                Debug.LogError("No processes found. Did you delete or manually rename them?");
```